setup.py

```diff
@@ -55,6 +55,10 @@ def read_version(file_path="version.txt"):
     and platform.system() == "Darwin"
 )
 
+use_cpp_avx512 = os.getenv("USE_AVX512", "0") == "1"
```

Comment:
This name might not be intuitive. This flag actual decide building of CPP kernels or not.